UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Services/FssService.cs

```diff
@@ -554,17 +496,63 @@ private CreateBatchRequestModel CreateBatchRequestModel(Batch batchType)
                     createBatchRequest.Attributes.Add(new KeyValuePair<string, string>("Media Type", "Zip"));
                     break;
-                
+
                 default:
                     break;
             };
             return createBatchRequest;
         }
 
+        [ExcludeFromCodeCoverage]
+        private CreateBatchRequestModel AddBatchAttributesForPos(string currentWeek, string currentYear)
+        {
+            CreateBatchRequestModel createBatchRequest = new()
+            {
+                BusinessUnit = _fssApiConfiguration.Value.BusinessUnit,
+                ExpiryDate = DateTime.UtcNow.AddDays(28).ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture),
+                Acl = new Acl
+                {
+                    ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.PosReadUsers) ? new List<string>() : _fssApiConfiguration.Value.PosReadUsers.Split(","),
+                    ReadGroups = string.IsNullOrEmpty(_fssApiConfiguration.Value.PosReadGroups) ? new List<string>() : _fssApiConfiguration.Value.PosReadGroups.Split(","),
+                },
+                Attributes = new List<KeyValuePair<string, string>>
+                {
+                    new("Product Type", "AVCS"),
+                    new("Week Number", currentWeek),
+                    new("Year", currentYear),
+                    new("Year / Week", currentYear + " / " + currentWeek)
+                }
+            };
+            return createBatchRequest;
+        }
+
+        [ExcludeFromCodeCoverage]
+        private CreateBatchRequestModel AddBatchAttributesForAio(string currentWeek, string currentYear)
+        {
+            CreateBatchRequestModel createBatchRequest = new()
+            {
+                BusinessUnit = _fssApiConfiguration.Value.AioBusinessUnit,
+                ExpiryDate = DateTime.UtcNow.AddDays(28).ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture),
+                Acl = new Acl
+                {
+                    ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.AioReadUsers) ? new List<string>() : _fssApiConfiguration.Value.AioReadUsers.Split(","),
+                    ReadGroups = string.IsNullOrEmpty(_fssApiConfiguration.Value.AioReadGroups) ? new List<string>() : _fssApiConfiguration.Value.AioReadGroups.Split(","),
+                },
+                Attributes = new List<KeyValuePair<string, string>>
+                {
+                    new("Product Type", "AIO"),
+                    new("Week Number", currentWeek),
+                    new("Year", currentYear),
+                    new("Year / Week", currentYear + " / " + currentWeek)
+                }
+            };
+            return createBatchRequest;
+        }
+
         //Private Methods
         [ExcludeFromCodeCoverage]
         private CreateBatchRequestModel CreateBatchRequestModel(Batch batchType, ConfigQueueMessage configQueueMessage)
```

Comment:
To keep it consistent
```suggestion
        private CreateBatchRequestModel AddBatchAttributesForBess(Batch batchType, ConfigQueueMessage configQueueMessage)
```

Reply:
Resolved. Renamed as CreateBatchRequestModelForBess